examples/package.json

```diff
@@ -1,18 +1,23 @@
-
 {
   "name": "react-globalize-example",
+  "version": "0.4.1",
   "description": "Examples demonstrating use of the components",
   "main": "index.js",
   "devDependencies": {
-    "browserify": "^6.3.3",
-    "reactify": "^0.17.1"
+    "babel-core": "^6.25.0",
+    "babel-loader": "^7.1.1",
+    "babel-plugin-transform-class-properties": "^6.24.1",
+    "babel-preset-react": "^6.24.1",
+    "imports-loader": "^0.7.1",
+    "webpack": "^3.5.4"
   },
```

Comment:
Let's add react ^16.0.0 in devDeps..

Reply:
I'm really not good at this! Isn't this a problem if deps are still ^15?

Reply:
Yeap, we should loose that in dev by adding the `|| 16` :)